### PR TITLE
Specify the local repo in order to find the image

### PR DESCRIPTION
**Pull request by rob-baron**
This way openstack will pull from the local repo as opposed to trying to pull from somewhere else and failing.  The containers found in this job are build locally from the build configs.

### CODE REVIEW

File: k8s/overlays/nerc-ocp-infra/kustomization.yaml [hunk 1]
```diff
@@ -10,4 +10,5 @@ resources:
 
 patchesStrategicMerge:
   - patches/cj-xdmod-openstack-shred.yaml
+  - patches/cj-xdmod-ingestor.yaml
```
Comment (knikolla) [inline k8s/overlays/nerc-ocp-infra/kustomization.yaml:13]: This seems unrelated?

Comment (knikolla) [reply k8s/overlays/nerc-ocp-infra/kustomization.yaml:13]: Ah, had to refresh. Now I see it.

**Review by knikolla — APPROVED** [review]

File: k8s/overlays/nerc-ocp-infra/kustomization.yaml (hunk 1 above)
Comment (rob-baron) [reply k8s/overlays/nerc-ocp-infra/kustomization.yaml:13]: how so?
